Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2389,3 +2389,51 @@ def _is_status_code_valid(self, response, ok_codes=None):
 
 class DemistoException(Exception):
     pass
+
+
+def arg_to_boolean(value):
+    """
+        Boolean-ish arguments passed through demisto.args() could have type bool or type string.
+        This takes the guesswork out and will return a bool, regardless of value's type.
+        It will also return True for 'yes' and False for 'no'.
+
+        :param value: the value to evaluate
+        :type value: ``string|bool``
+
+        :return: a boolean representatation of 'value'
+        :rtype: ``bool``
+    """
+    if isinstance(value, bool):
+        return value
+    if isinstance(value, STRING_OBJ_TYPES):
+        if value.lower() in ['true', 'yes']:
+            return True
+        elif value.lower() in ['false', 'no']:
+            return False
+        else:
+            return_error('Argument does not contain a valid boolean-ish value')
```

Comment:
raise ValueError instead return_error. We are trying to reduce return_error, because return_error doing sys.exit(1)

Reply:
Done.  Also renamed function to argToBoolean() to be consistent with argToList().